test/parallel/test-stream-readable-removeListener.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+var assert = require('assert');
+var EE = require('events').EventEmitter;
```

Comment:
The `EventEmitter` at the end is not necessary, as `EventEmitter` is the one which is exported.